tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -130,10 +131,60 @@ Spawn Notebook
     [Documentation]  Start the notebook pod spawn and wait ${spawner_timeout} seconds (DEFAULT: 600s)
     [Arguments]  ${spawner_timeout}=600 seconds
     Click Button  Start server
-    Wait Until Page Contains  Starting server  60s
+    # Waiting for 60 seconds, since a long wait seems to redirect the user to the control panel
+    # if the spawn was successful
+    ${modal} =    Run Keyword And Return Status    Wait Until Page Contains
+    ...    Starting server    60s
+    IF  ${modal}==False
+        Log    message=Starting server modal didn't appear after 60s    level=ERROR
```

Comment:
@lugi0 @jgarciao @bdattoma, the error message shouldn't be after 60s, it should be after 5s, 10s max.

It's fine for the test to wait longer and do what's necessary not to crash when the popup doesn't appear this fast, but you can't have only green lights when this happens...

Reply:
this line would raise a red light since the level is set to ERROR. @lugi0 right?

Reply:
it will raise the red flag ... but only after `60s`. The popup should appear "immediately".